### PR TITLE
Update with changes to cardano-ledger-specs

### PR DESCRIPTION
**Pull request by nc6**
This merely updates to the latest version of cardano-ledger-specs

### CODE REVIEW

File: cardano-ledger/test/Test/Cardano/Chain/UTxO/Model.hs [hunk 1]
```diff
@@ -1,3 +1,4 @@
+{-# LANGUAGE BangPatterns #-}
```
Comment (intricate) [inline cardano-ledger/test/Test/Cardano/Chain/UTxO/Model.hs:1]:
```suggestion
{-# LANGUAGE BangPatterns      #-}
```

File: cardano-ledger/test/Test/Cardano/Chain/Block/Model.hs [hunk 2]
```diff
@@ -410,7 +410,7 @@ ts_prop_invalidTxWitsAreRejected =
     block <- sigGenChain NoGenDelegation NoGenUTxO NoGenUpdate env st
     let (_slot, _sgs, _h, utxoSt, _delegSt, _upiSt) = st
         utxoEnv = UTxOEnv { utxo0 = utxo, pps = pparams }
-    txWitsList <- tamperedTxWitsList utxoEnv utxoSt
+    txWitsList <- tamperedTxList utxoEnv utxoSt
```
Comment (intricate) [inline cardano-ledger/test/Test/Cardano/Chain/Block/Model.hs:413]: Should this binding perhaps be renamed or is it still accurate?

```suggestion
    txList <- tamperedTxList utxoEnv utxoSt
```

Comment (nc6) [reply cardano-ledger/test/Test/Cardano/Chain/Block/Model.hs:413]: There are still witnesses there

File: cardano-ledger/test/Test/Cardano/Chain/UTxO/Model.hs [hunk 3]
```diff
@@ -13,7 +14,7 @@ module Test.Cardano.Chain.UTxO.Model
   )
 where
 
-import Cardano.Prelude hiding (trace)
+import Cardano.Prelude hiding (trace, traceM, traceShow)
```
Comment (intricate) [inline cardano-ledger/test/Test/Cardano/Chain/UTxO/Model.hs]: It seems that these additional imports are redundant:

```suggestion
import Cardano.Prelude hiding (trace)
```

Comment (intricate) [inline cardano-ledger/test/Test/Cardano/Chain/UTxO/Model.hs:17]: It seems that these additional hidden symbols are redundant:

```suggestion
import Cardano.Prelude hiding (trace)
```

**Review by erikd — APPROVED** [review]

**Pull request merged**